src/machine/machinectl.c

```diff
@@ -1667,7 +1668,21 @@ static int enable_machine(int argc, char *argv[], void *userdata) {
                 goto finish;
         }
 
-        r = 0;
+        if (arg_now) {
+                _cleanup_strv_free_ char **new_args = NULL;
+
+                new_args = strv_new(streq(argv[0], "enable") ? "start" : "stop");
+                if (!new_args) {
+                        r = -ENOMEM;
+                        goto finish;
+                }
+
+                r = strv_extend_strv(&new_args, argv + 1, /* filter_duplicates = */ false);
+                if (r < 0)
+                        goto finish;
```

Comment:
also log_oom() here.

(strv_extend_strv() can only fail on ENOMEM, hence we can sloppily just say:

```c
r = strv_extend_strv(…)
if (r < 0) {
        log_oom();
        goto finish;
}